docs/coding-style.md

```diff
@@ -218,6 +285,15 @@ It can still be used when and where it makes sense. For instance, when a class h
 implementing generic interfaces (such as `IComparable`, `IDisposable`), it can make sense to have regions 
 for the implementation of these interfaces.
 
+## Spacing
+
+* Avoid spaces unless they bring clarity and help the reader understand logical groups. Prefer spaces over comments.
```

Comment:
By spaces you mean newlines? 
It's a bit confusing, someone might read this as using `new int[]{1,2,3}` instead of `new int[] { 1, 2, 3 }`

Reply:
yes, empty lines, I will update